crates/bevy_transform/src/lib.rs

```diff
@@ -5,169 +5,43 @@
     html_favicon_url = "https://bevyengine.org/assets/icon.png"
 )]
 
-pub mod commands;
 /// The basic components of the transform crate
 pub mod components;
+
+#[cfg(feature = "bevy-support")]
+pub mod commands;
+
+/// Transform related bundles
+#[cfg(feature = "bevy-support")]
+pub mod bundles;
+
+/// Transform related bundles
+#[cfg(feature = "bevy-support")]
```

Comment:
This shouldn't be feature-flagged: `bevy_math` is a lightweight wrapper over `glam` and is generally useful.

Reply:
Removed